lib/protoflow-blocks/doc/sys/write_socket.mmd

```diff
@@ -0,0 +1,9 @@
+block-beta
+    columns 4
+    Source space:2 WriteSocket
+    Source-- "input" -->WriteSocket
+
+    classDef block height:48px,padding:8px;
+    classDef hidden visibility:none;
+    class WriteSocket block
+    class Source hidden
```

Comment:
Make sure to newline-terminate all text files

Reply:
done

Reply:
The same problem as on the other PR, trailing whitespace which then leads to GitHub's warning:

<img width="699" alt="Screenshot 2024-11-19 at 5 55 34 PM" src="https://github.com/user-attachments/assets/85b7062d-b143-4f95-a27f-c0acce348ae9">

Reply:
I don't see any warnings in github now, fixed